Tests/StencilSwiftKitTests/JSExtensionTests.swift

```diff
@@ -0,0 +1,151 @@
+//
+// StencilSwiftKit
+// Copyright (c) 2017 SwiftGen
+// MIT Licence
+//
+
+import XCTest
+import StencilSwiftKit
+
+class JSExtensionTests: XCTestCase {
+
+  func template(_ templateString: String,
```

Comment:
You've added more tests, which is good, but you're still missing (mentioned before):
- given a list of tokens, parses a correct structure of nodes/filters and such
- given a structure of nodes and a context, renders the correct output

You're directly going from stencil template string to a render. We prefer to test this in 2 fases: parsing, and then rendering.